src/docs/release-notes/R16/R16.mdx

```diff
@@ -0,0 +1,23 @@
+### Documentation Site
+
+Added search capability to doc site
+
+### New Components
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we have added components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
+    [Scorecard](https://pxblue-components.github.io/angular-dev/?path=/info/components-score-card--readme), and [InfoListItem](https://pxblue-components.github.io/angular-dev/?path=/info/components-info-list-item--readme).
+-   For React Native, we have added a [Drawer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--drawer) component.
+
+### React Native Reusable Workflow
+
+Innersourced a reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native apps
```

Comment:
```suggestion
We have finished innersourcing a reusable [Authentication Workflow](https://github.com/pxblue/react-native-workflows) for React Native apps, allowing teams to use a common implementation across different projects.
```

Reply:
I've used this edited version but have taken out the React Native Auth workflow link since the repo is private and this'll fail our pipeline.